bin/app.rb

```diff
@@ -0,0 +1,21 @@
+require_relative 'music_album'
+
+class App
+  def initialize
+    @music_album = []
+    @genres = []
+  end
+
+  def add_music_album(name, on_spotify, publish_date)
+    music_album = MusicAlbum.new(name, on_spotify, publish_date)
+    @music_album << music_album
+  end
+
+  def list_music_albums
+    @music_album.each { |music_album| puts music_album }
```

Comment:
To be able to "puts" an object you need to implement the "to_s" special method inside the object. In this case, you need to add a to_s method to the `MusicAlbum` class.

This is the `to_s` method for `Book` class:

```ruby
  def to_s
    "(#{@id}) publisher: #{@publisher} - cover state: #{@cover_state} - publish date: #{@publish_date.to_date}"
  end
```